### DISCUSSION TITLE
How to output logging data on the JVM?

### DISCUSSION THREAD

**Discussion by wangtianying1111** [NTNU-IHB/FMI4j#147] [Q&A] [answered]
I’d be grateful if someone could give me some guidance on how to output logging data on the JVM

**Comment by markaren** [comment, marked as answer]
The library does not support this. The easiest way to go about something that could act as a solution would be for you do modify the native logging function to write to a file. Not ideal,of course. 
Getting the information directly back to the JVM on the other hand would involve defining a JNI callback function to be called by the native logger. This is not something I plan to do, however.
